rlclientlib/schema/v2/Event.fbs

```diff
@@ -9,12 +9,15 @@ table Event {
 }
 
 table BatchMetadata {
-    content_encoding: string;
+    content_encoding: string; //valid values: IDENTITY and DEDUP
+}
+
+table SerializedEvent {
```

Comment:
q:Where does the meta field go for SerializedEvent?

Reply:
There isn't one, all metadata lives in Event itself.